packages/remix-dev/compiler/plugins/serverBareModulesPlugin.ts

```diff
@@ -0,0 +1,87 @@
+import { builtinModules } from "module";
+import { isAbsolute, relative } from "path";
+
+import type { Plugin } from "esbuild";
+
+import { RemixConfig } from "../../config";
+
+export function serverBareModulesPlugin(
```

Comment:
Needs a good doc comment!

Reply:
Maybe call it the `serverExternalsPlugin`

Reply:
I think I'd like to keep it bare modules because it does more than just externalize.